packages/ui/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@prezly/theme-kit-ui",
+  "version": "0.0.1",
+  "description": "UI components for Prezly themes",
+  "keywords": ["prezly", "themes", "ui"],
+  "author": "Prezly Team",
+  "homepage": "https://github.com/prezly/theme-kit-js/tree/main/packages/ui#readme",
+  "license": "MIT",
+  "main": "build/index.js",
+  "files": ["build", "README.md"],
+  "types": "build/index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/prezly/theme-kit-js.git"
+  },
+  "scripts": {
+    "prebuild": "rimraf build/*",
+    "build": "microbundle build --tsconfig ./tsconfig.json --compress --format modern,cjs src/index.ts",
+    "dev": "microbundle watch --tsconfig ./tsconfig.json --format modern,cjs src/index.ts",
+    "build-storybook": "storybook build",
+    "storybook": "storybook dev -p 6006"
+  },
+  "bugs": {
+    "url": "https://github.com/prezly/theme-kit-js/issues"
+  },
+  "peerDependencies": {
+    "next": "^12.x || ^13.x",
```

Comment:
Why should plain UI components depend on Next? 🤔 

Also, if these components are intended to be used for new themes, it doesn't make sense to support Next 12 or React 17.

Reply:
Dropped support for Next and React 17